gossipd/test/run-extended-info.c

```diff
@@ -0,0 +1,790 @@
+{ fprintf(stderr, "towire_gossip_get_txout called!\n"); abort(); }
+/* Generated stub for towire_gossip_new_peer_reply */
+u8 *towire_gossip_new_peer_reply(const tal_t *ctx UNNEEDED, bool success UNNEEDED, const struct gossip_state *gs UNNEEDED)
+{ fprintf(stderr, "towire_gossip_new_peer_reply called!\n"); abort(); }
+/* Generated stub for towire_gossip_ping_reply */
+u8 *towire_gossip_ping_reply(const tal_t *ctx UNNEEDED, const struct node_id *id UNNEEDED, bool sent UNNEEDED, u16 totlen UNNEEDED)
+{ fprintf(stderr, "towire_gossip_ping_reply called!\n"); abort(); }
+/* Generated stub for towire_gossip_query_channel_range_reply */
+u8 *towire_gossip_query_channel_range_reply(const tal_t *ctx UNNEEDED, u32 final_first_block UNNEEDED, u32 final_num_blocks UNNEEDED, bool final_complete UNNEEDED, const struct short_channel_id *scids UNNEEDED)
+{ fprintf(stderr, "towire_gossip_query_channel_range_reply called!\n"); abort(); }
+/* Generated stub for towire_gossip_scids_reply */
+u8 *towire_gossip_scids_reply(const tal_t *ctx UNNEEDED, bool ok UNNEEDED, bool complete UNNEEDED)
+{ fprintf(stderr, "towire_gossip_scids_reply called!\n"); abort(); }
+/* Generated stub for towire_hsm_cupdate_sig_req */
+u8 *towire_hsm_cupdate_sig_req(const tal_t *ctx UNNEEDED, const u8 *cu UNNEEDED)
+{ fprintf(stderr, "towire_hsm_cupdate_sig_req called!\n"); abort(); }
+/* Generated stub for towire_hsm_node_announcement_sig_req */
+u8 *towire_hsm_node_announcement_sig_req(const tal_t *ctx UNNEEDED, const u8 *announcement UNNEEDED)
+{ fprintf(stderr, "towire_hsm_node_announcement_sig_req called!\n"); abort(); }
+/* Generated stub for towire_wireaddr */
+void towire_wireaddr(u8 **pptr UNNEEDED, const struct wireaddr *addr UNNEEDED)
+{ fprintf(stderr, "towire_wireaddr called!\n"); abort(); }
+/* Generated stub for wireaddr_eq */
+bool wireaddr_eq(const struct wireaddr *a UNNEEDED, const struct wireaddr *b UNNEEDED)
+{ fprintf(stderr, "wireaddr_eq called!\n"); abort(); }
+/* Generated stub for wire_sync_read */
+u8 *wire_sync_read(const tal_t *ctx UNNEEDED, int fd UNNEEDED)
+{ fprintf(stderr, "wire_sync_read called!\n"); abort(); }
+/* Generated stub for wire_sync_write */
+bool wire_sync_write(int fd UNNEEDED, const void *msg TAKES UNNEEDED)
+{ fprintf(stderr, "wire_sync_write called!\n"); abort(); }
+/* AUTOGENERATED MOCKS END */
+
+#if DEVELOPER
+/* Generated stub for memleak_remove_htable */
+void memleak_remove_htable(struct htable *memtable UNNEEDED, const struct htable *ht UNNEEDED)
+{ fprintf(stderr, "memleak_remove_htable called!\n"); abort(); }
+/* Generated stub for memleak_remove_intmap_ */
+void memleak_remove_intmap_(struct htable *memtable UNNEEDED, const struct intmap *m UNNEEDED)
+{ fprintf(stderr, "memleak_remove_intmap_ called!\n"); abort(); }
+#endif
+
+/* Generated stub for status_fmt */
+void status_fmt(enum log_level level UNNEEDED, const char *fmt UNNEEDED, ...)
+{
+}
+
+static const char *test_vectors[] = {
```

Comment:
Loading the test vectors from file would have allowed us to update them more easily in the future, but for now that's ok.

Reply:
There'll be a rework of test vectors in the spec sometime, and we can clean up how we treat them then.